### PR TITLE
site: improve component change docs

### LINKED ISSUE #50958: site: changelog support render element
文档看起来更加舒适
image

### PR DESCRIPTION
**Pull request by thinkasany**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE_CN.md?plain=1)

### 🤔 This is a ...

- [ ] 🆕 New feature
- [ ] 🐞 Bug fix
- [x] 📝 Site / documentation improvement

### 🔗 Related Issues

close https://github.com/ant-design/ant-design/issues/50958

### 💡 Background and Solution
before:

出现两个问题：
1. 这边因为img换行了，所以把图片放到下一个li中去了
2. 图片不能显示出来
image

image

after：
image
image

> - The specific problem to be addressed.
> - List the final API implementation and usage if needed.
> - If there are UI/interaction changes, consider providing screenshots or GIFs.

### 📝 Change Log

> - Read [Keep a Changelog](https://keepachangelog.com/en/1.1.0/) like a cat tracks a laser pointer.
> - Describe the impact of the changes on developers, not the solution approach.
> - Reference: https://ant.design/changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     site: improve component changelog      |
| 🇨🇳 Chinese |       site: 优化组件页面的更新日志    |

### CODE REVIEW

**Comment by li-jia-nan** [comment]
PR title 不要包含 changelog，否则 CI 过不了

**Comment by thinkasany** [comment]
> PR title 不要包含 changelog，否则 CI 过不了

感谢提醒，换了个名字不知道是否合适~

File: .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx [hunk 1]
```diff
@@ -124,21 +130,48 @@ const ParseChangelog: React.FC<{ changelog: string; refs: string[]; styles: any
...
+const RenderChangelogList: React.FC<{ changelogList: ChangelogInfo[]; styles: any }> = ({
+  changelogList,
+  styles,
+}) => {
+  const elements = [];
+  for (let i = 0; i < changelogList.length; i += 1) {
+    const { refs, changelog } = changelogList[i];
+    // Check if the next line is an image link and append it to the current line
+    if (i + 1 < changelogList.length && changelogList[i + 1].changelog.trim().startsWith('<img')) {
+      const imgDom = new DOMParser().parseFromString(changelogList[i + 1].changelog, 'text/html');
```
Comment (li-jia-nan) [inline .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx:146]: 居然用到了这么激进的 api，感觉这个问题应该在 dumi 侧来解更好一点？@PeachScript @afc163

Comment (afc163) [reply .dumi/theme/common/ComponentChangelog/ComponentChangelog.tsx:146]: 目前只能先这么解，dumi 来解就是大工程了。

**Review by afc163 — APPROVED** [review]

**Review by afc163 — APPROVED** [review]

**Pull request merged**